src/components/SettingDialog.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <q-dialog
+    maximized
+    persistent
+    class="setting-dialog"
+    v-model="modelValueComputed"
```

Comment:
Since `modelValueComputed` is hard to understand what it means, how about `dialogOpened`?